tests/test_recompute_navmesh.py

```diff
@@ -43,6 +43,13 @@ def test_recompute_navmesh(test_scene, sim):
     hab_cfg = examples.settings.make_cfg(cfg_settings)
     sim.reconfigure(hab_cfg)
 
+    # get the initial navmesh area. This test assumes default navmesh assets.
```

Comment:
One nit:  Can we move this to their own test?  It'll be easier to localize failures that way.